src/Tribe/Cache.php

```diff
@@ -58,15 +57,27 @@ public function set_transient( $id, $value, $expiration = 0, $expiration_trigger
 	}
 
 	/**
-	 * @param string $id
-	 * @param string $expiration_trigger
+	 * Get cached data. Optionally set data if not previously set.
+	 *
+	 * @param string       $id                 The key for the cached value.
+	 * @param string       $expiration_trigger Optional. Hook to trigger cache invalidation.
+	 * @param string|array $callback_if_empty  Optional. Sets the cache value to the return of this callback if no value was already set.
+	 * @param mixed        $callback_args      Optional. Args passed to callback.
+	 * @param int          $expiration         Optional. Value of expiration if we have to set the cache value.
 	 *
 	 * @return mixed
 	 */
-	public function get( $id, $expiration_trigger = '' ) {
+	public function get( $id, $expiration_trigger = '', $callback_if_empty = null, $callback_args = array(), $expiration = 0 ) {
```

Comment:
Rather than `callback_if_empty` here I would use `default`; I'd add a check to see if `$default` `is_callable()` and either use that or use the return value of the callback. I would also flip the `expiration` and `callback_args` parameters to have a signature like this:
```php
public function get( $id, $expiration_trigger = '', $default = null,  $expiration = 0, $args = array())
```
The reason being that we might more frequently use the `expiration` parameter than the `args` (I'd rename those to decouple the parameter from the concept of a callback).